src/app/components/media/MediaParentComponent.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component, PropTypes } from 'react';
+import MediaComponent from './MediaComponent';
+import Can from '../Can';
+import Translation from '../translation/Translation';
+import config from 'config';
+
+class MediaParentComponent extends Component {
+  render() {
+
+    const extraComponents = [
+      <Can permissions={this.props.media.permissions} permission="create Dynamic">
+        <Translation annotated={this.props.media} annotatedType="ProjectMedia" />
+      </Can>
+    ];
+
+    return config.appName === 'bridge' ? (
+      <MediaComponent {...this.props} extras={extraComponents} />
```

Comment:
Not exactly on this line, but on this file yes, to tell that only Bridge has the `Translation` as an extra component (the condition could be used to define `extraComponents`)